js/nativeplot/Example.svelte

```diff
@@ -0,0 +1 @@
+<div></div>
```

Comment:
You don't need to add an `Example.svelte` file since this component is not going to be used as an input